actioncable/lib/action_cable/subscription_adapter/redis.rb

```diff
@@ -150,6 +150,9 @@ def ensure_listener_running
 
                 conn = @adapter.redis_connection_for_subscriptions
                 listen conn
+              rescue Redis::BaseConnectionError
```

Comment:
In local dev I've found this needs to be 

```suggestion
              rescue ::Redis::BaseConnectionError
```

Or else you get a `uninitialized constant ActionCable::SubscriptionAdapter::Redis::BaseConnectionError`

Reply:
You're right. Fixed.